blazor/rich-text-editor/import-export.md

```diff
@@ -44,13 +44,13 @@ The Rich Text Editor allows you to load an external HTML file in the editor cont
 {% endhighlight %}
 {% endtabs %}
 
-![Import to HTML file](./images/blazor-import-html.png)
+![Blazor RichTextEditor import to HTML file](./images/blazor-richtexteditor-import-html.png)
 
 > [View Sample in GitHub](https://github.com/SyncfusionExamples/import-html-file-to-blazor-rich-text-editor).
 
 ## Import to RTF file
 
-You can import the RTF file into the editor by using file uploader component, and get the RTF file content from uploader success event. Then, you can able to import the RTF values to the editor.
+You can import the RTF file into the editor by using the file uploader component, and you can get the RTF file content from the uploader success event. Then, you can import the RTF values into the editor.
```

Comment:
Import the RTF file into the editor by using the file uploader component and get the RTF file content from the uploader success event. Then, import the RTF values into the editor.